bindings/py/tests/sparse_matrix_test.py

```diff
@@ -162,6 +162,70 @@ def test_setAllNonZeros(self):
         error('setAllNonZeros, out of order')
 
 
+  def test_nNonZerosPerRow_allRows(self):
+    for name, denseMatrix, expected in (
```

Comment:
I find this makes things more confusing then just writing out the two lines for each case.